apis/farms/src/handler.ts

```diff
@@ -70,7 +70,7 @@ export async function saveFarms(chainId: number, event: ScheduledEvent | FetchEv
   try {
     const isTestnet = farmFetcher.isTestnet(chainId)
     const farmsConfig = await (await fetch(`${farmConfigApi}/${chainId}.json`)).json<SerializedFarmConfig[]>()
-    let lpPriceHelpers: SerializedFarmConfig[] = []
+    const lpPriceHelpers: SerializedFarmConfig[] = []
```

Comment:
it will throw eslint `Cannot assign to 'lpPriceHelpers' because it is a constant.ts`